admin/templates/common_auth/emails/invitation_email.html

```diff
@@ -0,0 +1 @@
+{% extends 'registration/password_reset_email.html' %}
```

Comment:
What's the purpose of using an `extends` here? Why not have the template code directly in this file? I don't find `password_reset_email` used anywhere else in the codebase, and I don't see any other files in `admin/templates/common_auth/emails/` that would establish a precedent.